api/views.py

```diff
@@ -80,9 +80,9 @@ def post(self, request, pk):
     new_lessonstudent = student.lessonstudent_set.create(
       lesson_id=request.data['lesson'],
       score=request.data['score'],
-      mood=request.data['mood']
+      mood=request.data['mood'],
+      mood_analyzer=watson_service.analyze_tone(request.data['mood'])
```

Comment:
This watson service acts as a module almost. I didn't have to make it an object to call on the method

Reply:
I like this feature of python, it really makes everything modulable. Nice use of it.